x11-misc/sunflower/sunflower-0.3.61.ebuild

```diff
@@ -0,0 +1,80 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+inherit desktop gnome2-utils python-r1 xdg-utils
+
+MY_PN="Sunflower"
+DESCRIPTION="Small and highly customizable twin-panel file manager with plugin-support"
+HOMEPAGE="https://gitlab.com/MeanEYE/Sunflower
+	http://sunflower-fm.org/"
+SRC_URI="http://sunflower-fm.org/pub/sunflower-0.3-61.tgz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+REQUIRED_USE="${PYTHON_REQUIRED_USE}"
+
+DEPEND="${PYTHON_DEPS}"
+RDEPEND="${DEPEND}
+	>=dev-python/pygtk-2.15.0[${PYTHON_USEDEP}]
+	>=dev-python/notify-python-0.1[${PYTHON_USEDEP}]
+	dev-python/chardet[${PYTHON_USEDEP}]
+	gnome-base/librsvg:2"
```

Comment:
I'm sorry if I already asked this but how is librsvg used here? The pure-RDEP looks suspicious.